research/backend/restful/register.py

```diff
@@ -29,11 +29,15 @@ def post(self):
         else:
             hashed = bcrypt.hashpw(password, bcrypt.gensalt())
 
-        # Insert in RedisDB
-        dbmanager.REDIS_DB.hmset( 'username:' + username, {
```

Comment:
Stick your first parameter and opening parenthesis together.

Reply:
I fixed it, thanks!